shared/ui/Stream/PullRequests/Bitbucket/MergeScreen.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import { api } from "../../../store/providerPullRequests/thunks";
+import { useAppDispatch } from "@codestream/webview/utilities/hooks";
+import { FetchThirdPartyPullRequestPullRequest } from "@codestream/protocols/agent";
+
+import { Dialog } from "../../../src/components/Dialog";
+import Button from "../../Button";
+import { InlineMenu } from "@codestream/webview/src/components/controls/InlineMenu";
+import { Modal } from "../../Modal";
+import { WarningBox } from "../../WarningBox";
+
+interface Props {
+	pr: FetchThirdPartyPullRequestPullRequest;
+	onClose: Function;
+}
+
+export const MergeScreen = (props: Props) => {
+	const pr = props;
+	const dispatch = useAppDispatch();
+	const [mergeMessage, setMergeMessage] = useState("");
```

Comment:
Empty string `""` can cause sneakly problems - you have to handle "" in addition to null and undefined - better to use undefined when value not present

```suggestion
  const [mergeMessage, setMergeMessage] = useState<string>();
```